docs/stages.adoc

```diff
@@ -110,6 +110,63 @@ public SELF comma() {
 }
 ----
 
+=== Filler Words
+
+Filler words can be used to build a common and reusable vocabulary for your tests. This enables you to write
+beautifully fluent tests whilst at the same time keeping your stage methods clear and concise.
+
+The `@FillerWord` annotation can also be added to stage methods where you want to maintain the continuation of
+of sentences in reports.
+
+For example, given the following filler words:
+
+[source,java]
+----
+include::{sourcedir}/userguide/ExtendedVocabularyTest.java[tags=fillerWords,indent=0]
+----
+
+You can write test stages such as:
+
+[source,java]
+----
+include::{sourcedir}/userguide/ExtendedVocabularyTest.java[tags=givenFiller,indent=0]
+----
+
+This would generate the following report:
+
+----
+Given the ingredients
+      an egg
+      some milk
+      and the ingredient flour
+----
+
+Filler words can also be used to add punctuation to your tests. Filler words marked as punctuation should be
+complemented with the `@As` annotation.
+
+For example, given the following filler word:
+
+[source,java]
+----
+include::{sourcedir}/userguide/ExtendedVocabularyTest.java[tags=punctuation,indent=0]
+----
+
+You could write test stages such as:
+
+[source,java]
+----
+include::{sourcedir}/userguide/ExtendedVocabularyTest.java[tags=givenPunctuation,indent=0]
+----
+
+This would generate the following report:
+
+----
+Given a clean worksurface, a bowl and the ingredients
+     an egg
+     some milk
+     the ingredient flour
```

Comment:
(optional) you might a `.` here and in the test just have a better visibility. I actually had to look for the punctuation. Might be an due to the page not being rendered :smile:

Reply:
I've refined the punctuation concept even further. You may now use punctuation to end an sentence. Hopefully this example conveys the use of punctuation more clearly now.

Reply:
```
   Given a clean worksurface, a bowl and the ingredients:
         an egg
         some milk
         the ingredient flour
```
I think it looks quite smart! :smiley: